dbt_subprojects/dex/models/automated_trades/dex_automated_trades_unmapped.sql

```diff
@@ -0,0 +1,62 @@
+{{ config(
+    schema = 'dex'
+    , alias = 'automated_trades_unmapped'
+    , partition_by = ['blockchain', 'block_date']
+    , materialized = 'incremental'
```

Comment:
are you materializing this one (while mapped is not) because the `NOT EXISTS` is slow to run? why one view and not the other?